docs/guides/beginnerCLI.rst

```diff
@@ -0,0 +1,276 @@
+
+.. _VS Code: https://code.visualstudio.com 
+.. _Vim: https://www.vim.org/download.php 
+.. _Sublime: https://www.sublimetext.com 
+
+2. Locate the path to the and open the config.yaml file. You may notice that when you search, that the file does not exist, even though the path does. To fix this, we can manually create the file within the text editor. I am using VS Code in this example. Hover over *File* in the top left corner of the window and select *Open Folder* from the drop down. Select the location that the ``beet config -p`` command returned. Once there, hover over the folder name, and select the little page and plus icon next to it. This will create the new file. We will name this file *config.yaml*. Double check that the config.yaml and the library.db are in the *same* folder. 
+
+    .. image:: ../_static/Images/config.png
+        :width: 600
+
+The file will start out empty, but here's a good place to start::
+
+    directory: ~/music
+    library: ~/data/musiclibrary.db
+
+3. Change that first path to a directory where you'd like to keep your music. Then,
+for ``library``, choose a good place to keep a database file that keeps an index
+of your music library. (The config's format is `YAML`_. You'll want to configure your
+text editor to use spaces, not real tabs, for indentation. Also, ``~`` means
+your home directory in these paths)
+
+The default configuration assumes you want to start a new organized music folder
+(that ``directory`` above) and that you'll *copy* cleaned-up music into that
+empty folder using beets' ``import`` command (see below). But you can configure
+beets to behave many other ways:
+
+* Start with a new empty directory, but *move* new music in instead of copying
+  it (saving disk space). Put this in your config file::
+
+        import:
+            move: yes
+
+* Keep your current directory structure; importing should never move or copy
+  files but instead just correct the tags on music. Put the line ``copy: no``
+  under the ``import:`` heading in your config file to disable any copying or
+  renaming. Make sure to point ``directory`` at the place where your music is
+  currently stored.
+
+* Keep your current directory structure and *do not* correct files' tags: leave
+  files completely unmodified on your disk. (Corrected tags will still be stored
+  in beets' database, and you can use them to do renaming or tag changes later.)
+  Put this in your config file::
+
+        import:
+            copy: no
+            write: no
+
+  to disable renaming and tag-writing.
+
+By following this base config set up, your config file should
+look similarly to this.
+
+    .. image:: ../_static/Images/configBase.png
+        :width: 800
+
+There are approximately six million other configuration options you can set
```

Comment:
A joke is nice, but maybe not with this, just in case it scares them off about exploring more configuration options.